src/notification.scss

```diff
@@ -29,315 +31,235 @@ $block: #{$fd-namespace}-notification;
-
-    color: $fd-notification-title-color;
-    padding: 1rem 0;
-    margin-right: 1rem;
-    font-weight: 700;
-    width: 100%;
-
-    @include fd-rtl() {
-      margin-right: 0;
-      margin-left: 1rem;
-    }
-
-    &:first-child {
-      padding-left: 1rem;
+      display: flex;
+      align-items: baseline;
+      border-bottom: $fd-notification-border;
+      padding: 0.75rem 0.5rem 0.25rem 1rem;
 
       @include fd-rtl() {
-        padding-left: 0;
+        padding-left: 0.5rem;
         padding-right: 1rem;
       }
     }
-  }
-
-  &__close {
-    position: relative;
-    top: 0.45rem;
-    right: 0.45rem;
 
     @include fd-rtl() {
-      right: -0.45rem;
+      padding-left: 0;
+      padding-right: 0;
     }
   }
 
-  // ------------------------ BODY ------------------------
-  &__body {
+  &__content {
     @include fd-reset();
 
-    display: flex;
-    align-items: flex-end;
     width: 100%;
-  }
+    margin-left: 0.75rem;
+    overflow: auto;
 
-  &__content {
-    @include fd-reset();
+    @include fd-rtl() {
+      margin-right: 0.75rem;
+      margin-left: 0;
+    }
 
-    display: flex;
-    padding: 1rem;
-    width: 60%;
-  }
+    &:first-child {
+      margin-left: 0;
 
-  &__avatar {
-    @include fd-reset();
+      @include fd-rtl() {
+        margin-right: 0;
+      }
+    }
 
-    margin-right: 1rem;
-    min-width: 2rem;
-    min-height: 2rem;
+    .#{$block}__title {
+      @include fd-reset();
+      @include line-clamp(2);
 
-    @include fd-rtl() {
-      margin-right: 0;
-      margin-left: 1rem;
+      font-weight: bold;
+      font-size: var(--sapFontHeader6Size);
+      max-height: 2.5rem;
     }
-  }
 
-  &__text {
-    @include fd-reset();
+    .#{$block}__paragraph {
+      @include fd-reset();
+      @include line-clamp(2);
 
-    width: 100%;
-    display: flex;
-    flex-direction: column;
-    padding-right: 1rem;
-  }
+      padding-top: 0.5rem;
+      max-height: 2.75rem;
+    }
 
-  &__description {
-    @include fd-reset();
-    @include multiline-truncate($lineHeight: 1.45em, $lineCount: 2, $backgroundColor: $fd-notification-background-color);
+    .#{$block}__footer {
+      @include fd-reset();
+      
+      padding-top: 0.5rem;
 
-    color: $fd-notification-content-text-color;
-  }
+      &--author {
+        @include fd-truncate();
 
-  &__metadata {
-    @include fd-reset();
+        display: inline-block;
+        max-width: 40%;
+        vertical-align: middle;
+      }
 
-    color: $fd-notification-content-text-color;
-    margin-top: 0.375rem;
+      &--separator {
+        margin: 0 0.5rem;
+      }
+    }
   }
 
-  // ------------------------ FOOTER ------------------------
-  &__footer {
+  &__header {
     @include fd-reset();
 
-    width: 40%;
     display: flex;
-    align-items: center;
-    justify-content: space-between;
-    padding: 0 0.5rem 0.5rem 1rem;
-
-    & > * {
-      &:not(:last-child) {
-        margin: 0.25rem;
-      }
-    }
   }
 
   &__actions {
```

Comment:
This should have `margin-left: 1rem;`
![image](https://user-images.githubusercontent.com/26483208/93187324-fbef8480-f73f-11ea-987c-0b0e9ebe3a6c.png)
![image](https://user-images.githubusercontent.com/26483208/93187347-ff830b80-f73f-11ea-94ca-61086fce68e0.png)